packages/unit/src/index.ts

```diff
@@ -6,3 +6,12 @@ export type {
   SolitaryTestBedBuilder,
   SociableTestBedBuilder,
 } from '@suites/core.unit';
+
+// Re-export commonly used types from testing framework-specific packages
+export type {
+  Mocked,
+  JestMocked,
+  VitestMocked,
+  SinonMocked,
+  StubbedInstance,
+} from './types.doubles.ts';
```

Comment:
> Adding those types here seems problematic since the unit package isn't linked to any 
> dependency injection or mocking library adapter (e.g. @suites/types.lib). Is there a specific reason for including them?

See to my previous comment